chef/cookbooks/neutron/recipes/common_config.rb

```diff
@@ -150,7 +150,8 @@
       :rootwrap_bin =>  node[:neutron][:rootwrap],
       :use_namespaces => true,
       :allow_overlapping_ips => neutron[:neutron][:allow_overlapping_ips],
-      :dvr_enabled => neutron[:neutron][:use_dvr]
+      :dvr_enabled => neutron[:neutron][:use_dvr],
+      :network_nodes_count => get_network_nodes.count
```

Comment:
Have you tried using it outside of the template? This should work there. So you just need to assign the results to a variable and use the variable here.